app/jobs/upvs/fetch_services_with_forms_list_job.rb

```diff
@@ -60,4 +58,17 @@ def each_row_as_attributes(csv_file, csv_options)
       )
     end
   end
+
+  def check_row_attributes(row)
+    if row.has_key?("IdServiceInstance,ExternalCode")
+      row["IdServiceInstance"], row["ExternalCode"] = row["IdServiceInstance,ExternalCode"].split(',')
+      row.delete("IdServiceInstance,ExternalCode")
+    end
+
+    if row.has_key?("FormURL;;")
+      row["FormURL"] = row.delete("FormURL;;")
+      row["FormURL"] = row["FormURL"].gsub(";;", "") if row["FormURL"]
+    end
+  end
```

Comment:
Nie je toto pozostatok zo zleho formatovania z minulosti?

Reply:
ano ano, mozem to dat prec, nechal som to pre pripad, ze keby sme to mozno chceli v buducnosti pouzit s nejakou upravou

Reply:
Budeme pevne dufat, ze uz nikdy nebude tak rozbite to CSV, ako bolo nedavno 😄